.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9", "3.10"]
```

Comment:
Don't we typically include 3.8 as well?

Reply:
We just need the latest 3 versions.

Reply:
We usually support 3.8 as well, but there may have been some build issues with 3.8? I can't remember the specifics.